src/Altinn.Platform/Altinn.Platform.Storage/UnitTest/TestingControllers/InstancesControllerTests.cs

```diff
@@ -276,6 +276,63 @@ public async void Delete_ReponseIsDeny_ReturnsStatusForbidden()
                 Assert.Equal(HttpStatusCode.Forbidden, response.StatusCode);
             }
 
+            /// <summary>
+            /// Test case: App owner tries to hard delete an instance
+            /// Expected: Returns success and deleted instance
+            /// </summary>
+            [Fact]
+            public async void Delete_OrgHardDeletesInstance_DeletedInstanceIsReturned()
```

Comment:
DeletedInstanceIsReturned could be more specific. For example: ReturnedInstanceHasStatusBothSoftAndHardDeleted.